### ISSUE TITLE
Twine dependency in 1.0.7

### ISSUE THREAD

**Issue by bluefish6** [csernazs/pytest-httpserver#250]
Hi, I'm just updating from 1.0.6 to 1.0.7 in my project and I have noticed that poetry wants to install a LOT of additional dependencies. I've tracked it down to the fact that https://pypi.org/pypi/pytest-httpserver/1.0.7/json shows:
```
        "requires_dist":
        [
            "Werkzeug (>=2.0.0)",
            "twine (>=4.0.2,<5.0.0)"                  <---- wait, what's this?
        ],
        "requires_python": ">=3.8,<4.0",
        "summary": "pytest-httpserver is a httpserver for pytest",
        "version": "1.0.7",
```
However, there is absolutely no mention of `twine` in the whole code repository... Is this some issue with packaging and uploading to pypi? It would be great to remove this dependency, given that it's not used. While `werkzeug` has only 1 dependency (markupsafe), `twine` has 27 of these...

**Comment by csernazs** [comment]
Hi there,

This has been added by mistake, I will fix it and release a new version.

Thanks for the bug report, 

Zsolt

**Comment by csernazs** [comment]
hi @bluefish6 

This has been fixed in 1.0.8 with #252, released to pypi.
I checked meta.json and it is ok.

I'm sorry for the mistake, I did poetry add twine to upload the release (for no reason, poetry can publish it :facepalm: ) but I did a `poetry build` before `poetry publish` also, so not just I published a bad release, it was published from an uncommitted tree.

Feel free to reopen this if something still not good.

Thanks for the report,
Zsolt
